op-devstack/presets/minimal.go

```diff
@@ -68,8 +68,10 @@ func NewMinimal(t devtest.T) *Minimal {
 		Wallet:        dsl.NewHDWallet(t, devkeys.TestMnemonic, 30),
 		Faucet:        dsl.NewFaucet(l2.Faucet(match.Assume(t, match.FirstFaucet))),
 	}
-	out.FaucetL1 = dsl.NewFaucet(out.L1Network.Escape().Faucet(match.Assume(t, match.FirstFaucet)))
-	out.FunderL1 = dsl.NewFunder(out.Wallet, out.FaucetL1, out.L1EL)
-	out.Funder = dsl.NewFunder(out.Wallet, out.Faucet, out.L2EL)
+	if out.Faucet != nil {
```

Comment:
generally  speaking, we want presets to either:
- apply, and then all components are provided
- or not apply, and then the test bails

If we want a preset without a faucet that's totally fine, but it should be a new one then

Reply:
happy to rename the presets :)
There's indeed a few things in there that don't qualify as "minimal".

But I really don't think having anything optional in a preset is actually helping

Reply:
Coolio, that can be tracked later as it's not a necessity for this PR per se.